<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="robot_name" default="robot"/>
  <arg name="agent_type" default="ground"/>
  <arg name="navigation_type" default="direct_pose_command"/>
  <arg name="localization_topic" default="pose"/>
  <arg name="scenario_file" default="$(find ltl_core)/config/ltl_plan_parameters.yaml"/>

  <group ns="$(arg robot_name)">
    
    <!-- Load parameters -->
    <rosparam command="load" file="$(arg scenario_file)" />

    <node name="ltl_planner" pkg="ltl_core" type="ltl_planner.py" output="screen"/>
      <param name="robot_name" type="string" value="$(arg robot_name)" />
      <param name="agent_type" type="string" value="$(arg agent_type)" />
      <param name="navigation_type" type="string" value="$(arg navigation_type)" />
	  <param name="localization_topic" type="string" value="$(arg localization_topic)" />
          <param name="scenario_file" type="string" value="$(arg scenario_file)" />

  </group>

</launch>
